regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
We need to define a specific view for the sub view header part, because the current preamble and comment views can't reach it because it's out of DOM scope (`.content-header` comes before `#content-wrapper`). So this particular view handles the click actions of the read/write tabs and will trigger the appropriate events in the general preamble view.